cg/meta/store/mip_dna.py

```diff
@@ -0,0 +1,119 @@
+"""Builds MIP DNA bundle for linking in Housekeeper"""
+import logging
+from pathlib import Path
+
+import ruamel.yaml
+
+from cg.constants import TAGS
+from cg.exc import AnalysisNotFinishedError, BundleAlreadyAddedError
+from cg.meta.store.base import (
+    reset_case_action,
+    add_new_analysis,
+    get_files,
+)
+
+LOG = logging.getLogger(__name__)
+
+
+def gather_files_and_bundle_in_housekeeper(config_stream, hk_api, status):
```

Comment:
Is this one not unnecessary now that you have it here: `from cg.meta.store.base import gather_files_and_bundle_in_housekeeper`

Reply:
The whole mip_dna.py module should not have been pushed, removed in https://github.com/Clinical-Genomics/cg/pull/621/commits/b1386ee141f04e6910548d78af68a10a872f7c1b

Reply:
To add to that, mip_rna.py is also redundant and should be removed. It is however used as an import in cg/apps/tb/add.py but I'm almost sure that's not used anymore. Still looking into that though.